### ISSUE TITLE
Simple way to remove paired devices

### ISSUE THREAD

**Issue by lucaskoehler** [madchicken/homebridge-zigbee-nt#43]
I can‘t find a way to remove a paired device. 
Even after reinstalling the zigbee plugin all devices remain in homebridge.
A button vor something like that whould help a lot.
Thanks!

**Comment by madchicken** [comment]
Yeah, I know. Removing accessories is not easy ATM. Unfortunately I don't have much time to implement a decent web UI to do that in this period, since I am really busy with my real job. Stay tuned, it's definitely something I want to add

**Comment by lucaskoehler** [comment]
Thanks for the information! Looking forward to your next updates 😊
Is there currently a kind of workaround to manually remove devices?

**Comment by madchicken** [comment]
You need to edit the zigbee database file, that should be under your homebridge configuration folder. It's a JSON file, just remove from there the device you don't want anymore

**Comment by mrdc** [comment]
@madchicken 
> You need to edit the zigbee database file, that should be under your homebridge configuration folder. It's a JSON file, just remove from there the device you don't want anymore

Unfortunately, this method doesn't work, even removing zigBee.db make no difference - devices are not deleted:
```
[12/11/2020, 09:14:07] [ZigBee] Loading accessory from cache: zigbee:permit-join
[12/11/2020, 09:14:07] [ZigBee] Loading accessory from cache: zigbee:touchlink
[12/11/2020, 09:14:07] [ZigBee] Loading accessory from cache: 0x00158d000xxxxxxx
[12/11/2020, 09:14:07] [ZigBee] Configured port for ZigBee dongle is /dev/ttyACM0
[12/11/2020, 09:14:07] [ZigBee] Initializing ZigBee controller on port /dev/ttyACM0 and channels 11, 25
[12/11/2020, 09:14:10] [ZigBee] ZigBee platform initialized @ 0x00124b001xxxxxxx
[12/11/2020, 09:14:10] [ZigBee] Reuse accessory from cache with uuid 6895c160-ba25-497f-b805-38071c751c56 and name zigbee:permit-join
[12/11/2020, 09:14:10] [ZigBee] Zigbee: disabling joining new devices.
[12/11/2020, 09:14:10] [ZigBee] PermitJoin accessory successfully registered
[12/11/2020, 09:14:10] [ZigBee] Reuse accessory from cache with uuid 4994893c-ea77-4292-a4ad-09e1f36f7099 and name zigbee:touchlink
[12/11/2020, 09:14:10] [ZigBee] Touchlink accessory successfully registered
[12/11/2020, 09:14:10] [ZigBee] Successfully started ZigBee service
```
Looks like devices are loaded from `cachedAccessories`

**Comment by madchicken** [comment]
Yes, that's right, you also need to remove the accessory from the homebridge cache, it can be done from the UI.
Btw, I am adding a web interface to work with devices, I should be able to release it in few days (with minimal functionalities)

**Comment by madchicken** [comment]
Fixed by #55
